### PR TITLE
Capstone2: Movie-website

### PR DESCRIPTION
**Pull request by GedeonTS**
## Movie-website

In this project with my partner @Lembani, we implemented a movie app using TVmaze API and capstone API for recording the users' actions. You can comment and like any movie you see through this app.

For the demo, Please Click [Here](https://drive.google.com/file/d/1THVuX3gwaVGB_qzDHzLvelM7QkA4SbDe/view?usp=sharing)!

your review is welcome :smile:

### CODE REVIEW

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @GedeonTS and @Lembani , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design
✔️  Good PR and Description


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/modules/header.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const headerElement = document.querySelector('.header');
+const nav = () => {
+  const navElement = `
+  <nav class="main-nav">
+    <a class="brand" href="#">Movie Website</a>
+    <ul class="nav-items">
+      <li class="nav-item"><a href="#" class="navlink">Home</a></li>
+      <li class="nav-item"><a href="#" class="navlink">Categories</a></li>
+    </ul>
+  </nav>
```
[the remark this answers is not in the document]
Comment (Lembani) [reply src/modules/header.js:10]: Hi @danmainah, thanks for the feedback. We will get right on to fix the issue. 🙏🏾

[the remark this answers is not in the document]
Comment (danmainah) [reply src/modules/header.js:10]: @Lembani  Great 👊 👊

**Review by zhadier — APPROVED** [review]
## Hi @GedeonTS & @Lembani :wave:  


In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



## Good luck and keep rocking! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by Lembani** [comment]
Hi @zhadier, thanks a lot for approving our PR. We really appreciate it. 💯

**Pull request merged**